python/GafferUITest/NumericPlugValueWidgetTest.py

```diff
@@ -56,7 +56,7 @@ def test( self ) :
 		self.assertTrue( w.getPlug().isSame( n["f"] ) )
 		self.assertTrue( isinstance( w.numericWidget().getValue(), float ) )
 
-		w = GafferUI.NumericPlugValueWidget( plug = None )
+		w = GafferUI.NumericPlugValueWidget( plugs = [] )
```

Comment:
`{}` is a dictionary, not a set. But I've deliberately coded `PlugValueWidget.__init__()` and others to accept any sequence type because there's no reason we can't, and I expect that's how folks will use it in practice. I think having a bit of test coverage for this is a good thing.

Reply:
Sorry, true indeed (it always seems ambiguous that you can use `{...}` for set comprehensions). Sounds like a good reason to keep it like that.